phpunit.xml.dist

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd" 
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd"
          colors="true"
-         verbose="true">
+         verbose="true"
+         bootstrap="src/autoload.php">
```

Comment:
I think so. Without it, the tests won't run on a clean checkout:

```
$ git clone https://github.com/google/recaptcha.git
Cloning into 'recaptcha'...
remote: Counting objects: 343, done.
remote: Total 343 (delta 0), reused 0 (delta 0), pack-reused 342
Receiving objects: 100% (343/343), 68.06 KiB | 0 bytes/s, done.
Resolving deltas: 100% (162/162), done.
Checking connectivity... done.

$ cd recaptcha/

$ phpunit
PHPUnit 4.3.1 by Sebastian Bergmann.

Configuration read from /home/mjo/tmp/recaptcha/phpunit.xml.dist


Fatal error: Class 'ReCaptcha\ReCaptcha' not found in /home/mjo/tmp/recaptcha/tests/ReCaptcha/ReCaptchaTest.php on line 38
```

I fixed the other two issues you mentioned.